test/MochaSpecRunner.html

```diff
@@ -0,0 +1,51 @@
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Mocha Tests</title>
+    <script type="text/javascript" src="lib/webcomponents.js"></script>
+    <link href="lib/mocha.css" rel="stylesheet" />
+    <link rel="import" href="../lab/polymer/components/my-menu-button.html">
+</head>
+
+<body>
+    <my-menu-button style="position: absolute; top: 0; right: 0" href="subpage.html">Distributed <strong>Content</strong></my-menu-button>
```

Comment:
Something stinks with the Polymer test, because if I don't do `bower install`, the tests are still passing:

![2017-04-04 15_44_08-mocha tests](https://cloud.githubusercontent.com/assets/566463/24659666/a5eb01a0-194d-11e7-8f44-9d66b4a737e6.png)